luaunit.lua

```diff
@@ -2074,6 +2094,9 @@ end
         ]]
 
         local expandedList = self.expandClasses( listOfNameAndInst )
+        if self.randomize then
+            M.private.randomizeTable( expandedList )
```

Comment:
The function is in the local scope, so there's no need to take the "private" route here - simply call `randomizeTable(expandedList)`.